pkg/cmd/server/api/types.go

```diff
@@ -46,8 +46,8 @@ type NodeConfig struct {
 	// DNSIP holds the IP
 	DNSIP string
 
-	// NetworkPluginName is a string specifying the networking plugin
-	NetworkPluginName string
+	// NodeNetworkConfig provides network plugin information
+	NodeNetworkConfig NodeNetworkConfig
```

Comment:
@liggitt we get a choice on this.  Do we want to bump to v2 and keep ourselves in sync or let the internal and external drift and be held together by conversions?